korma/src/commonMain/kotlin/com/soywiz/korma/geom/Anchor.kt

```diff
@@ -3,7 +3,7 @@ package com.soywiz.korma.geom
 import com.soywiz.korma.interpolation.Interpolable
 import com.soywiz.korma.interpolation.interpolate
 
-data class Anchor(val sx: Double, val sy: Double) : Interpolable<Anchor> {
+data class Anchor(var sx: Double, var sy: Double) : Interpolable<Anchor> {
```

Comment:
Anchor has immutable fields because it has a companion object with default values. This have several consequences:
* Might cause issues on Kotlin/Native, by having global mutable state
* Some might accidentally mutate one of the predefined instances, causing hard to catch bugs

Making Anchor mutable, would require changing this. In the case of the mutable `Matrix` class, instead of providing a global identity matrix, I'm adding a method to set a specific matrix to an identity one.

In this case, changing this would break existing code so we should keep this specific class immutable

Reply:
All right! I will change it back to immutable. In fact I had some doupts that this change might break something else :)